wagtail/admin/templates/wagtailadmin/pages/index.html

```diff
@@ -4,12 +4,11 @@
 {% block bodyclass %}page-explorer {% if ordering == 'ord' %}reordering{% endif %}{% endblock %}
 
 {% block content %}
-    <header class="w-header w-header--merged w-header--no-border">
-        <h1 class="visuallyhidden">Explorer</h1>
-
-        {% breadcrumbs parent_page 'wagtailadmin_explore' url_root_name='wagtailadmin_explore_root' %}
-    </header>
-
+    {% page_permissions parent_page as page_perms %}
+    <div class="w-sticky w-top-0 w-z-header">
+        {% include 'wagtailadmin/pages/page_listing_header.html' with title=parent_page.get_admin_display_title %}
```

Comment:
Let’s be explicit if this is what we need `page_perms` for (though I realise you’re just following other examples that aren’t so good).

```suggestion
        {% include 'wagtailadmin/pages/page_listing_header.html' with title=parent_page.get_admin_display_title page_perms= page_perms %}
```